Source/AtomicWebView/WebTexture2D.cpp

```diff
@@ -74,6 +75,52 @@ class WebTexture2DPrivate : public CefRenderHandler
         popupRectOriginal_ = rect;
         popupRect_ = GetPopupRectInWebView(popupRectOriginal_);
     }
+    void OnCursorChange(CefRefPtr<CefBrowser> browser, CefCursorHandle cursor, CursorType type, const CefCursorInfo& custom_cursor_info) OVERRIDE
+    {
+       SDL_Cursor *c;
+
+       SDL_SystemCursor id = SDL_SYSTEM_CURSOR_ARROW;
+
+       switch(type)
+       {
+       case CT_IBEAM:
+           id = SDL_SYSTEM_CURSOR_IBEAM;
+           break;
+       case CT_WAIT:
+           id = SDL_SYSTEM_CURSOR_WAIT;
+           break;
+       case CT_CROSS:
+           id = SDL_SYSTEM_CURSOR_CROSSHAIR;
+           break;
+       case CT_HAND:
+           id = SDL_SYSTEM_CURSOR_HAND;
+           break;
+       case CT_NORTHEASTSOUTHWESTRESIZE:
+           id = SDL_SYSTEM_CURSOR_SIZENESW;
+           break;
+       case CT_NORTHWESTSOUTHEASTRESIZE:
+           id = SDL_SYSTEM_CURSOR_SIZENWSE;
+           break;
+       case CT_EASTWESTRESIZE:
+           id = SDL_SYSTEM_CURSOR_SIZEWE;
+           break;
+       case CT_NORTHSOUTHRESIZE:
+           id = SDL_SYSTEM_CURSOR_SIZENS;
+           break;
+       case CT_MOVE:
+           id = SDL_SYSTEM_CURSOR_SIZEALL;
+           break;
+       case CT_NOTALLOWED:
+           id = SDL_SYSTEM_CURSOR_NO;
+           break;
+       case CT_POINTER:
+           break;
+       default:
+           break;
+       }
+       c = SDL_CreateSystemCursor(id);
+       SDL_SetCursor(c);
```

Comment:
Thanks, I am sure this makes the integrated WebView feel far more natural.  A couple notes, I believe cursors will leak without corresponding SDL_FreeCursor calls (https://wiki.libsdl.org/SDL_FreeCursor).  The cursors should probably be pre-created, reused, and freed at shutdown.  It would also make some sense that this is an optional feature as may not be using the system cursor, for example in fullscreen mode.